labs/a1.md

```diff
@@ -116,5 +116,38 @@ ab/abizer/some/path
 
 Hint: The easiest way to do this is with one very short command.
 
+## Question 4
+
+At some point, everyone has looked at a problem and thought to themselves:
+"Hey, I can do this in one line!"
+
+Lets find out if you can. I need to sort out some of my most listened to albums
+by making directories for each of their songs, specifically for my favorite artist Future.
+
+You need to fetch the list of albums and their respective artist from
+`https://raw.githubusercontent.com/0xcf/decal-labs/master/a1/albums.txt`.
```

Comment:
One of the albums has the `/` character which isn't allowed in a Linux filename.

Reply:
Yeah I didn't have a problem with it but its probably more safe to cut it. Sorry HONNE 😢